app/src/ui/history/selected-commits.tsx

```diff
@@ -116,6 +116,17 @@ export class SelectedCommits extends React.Component<
     this.props.dispatcher.changeFileSelection(this.props.repository, file)
   }
 
+  private onRowDoubleClick = (row: number) => {
+    const files = this.props.changesetData.files
+    const file = files[row]
+    if (!file) {
+      console.error('double click on row with no file - what?')
```

Comment:
This should be either `log.error(…)` or nothing. I think we can just remove it:
```suggestion
```

Reply:
Yea, agreed this is unnecessary 👍🏽